megamek/src/megamek/common/Compute.java

```diff
@@ -3160,72 +3196,72 @@ public static float getExpectedDamage(IGame g, WeaponAttackAction waa,
                         g.getEntity(waa.getTargetId()).getPosition());
                 //Convert AV to fDamage for bay weapons, fighters, etc
                 if (attacker.usesWeaponBays()){
-                	double av = 0;
-                	double threat = 1;
+                    double av = 0;
+                    double threat = 1;
                     for (int wId : weapon.getBayWeapons()) {
-                    	Mounted bayW = attacker.getEquipment(wId);
-                    	WeaponType bayWType = ((WeaponType) bayW.getType());
-                	//Capital weapons have a different range scale
+                        Mounted bayW = attacker.getEquipment(wId);
+                        WeaponType bayWType = ((WeaponType) bayW.getType());
+                    //Capital weapons have a different range scale
                         if (wt.isCapital()) {
-                        	// Capital missiles get higher priority than standard missiles:
+                            // Capital missiles get higher priority than standard missiles:
                             // damage plus a bonus for the critical hit threat they represent
-                        	threat = 12;
-                        	if (rangeToTarget > 50) {
-                			av = 0;
-                        	} else if (rangeToTarget > 40) {
-                			av += bayWType.getExtAV();
-                        	} else if (rangeToTarget > 25) {
-                			av += bayWType.getLongAV();
-                        	} else if (rangeToTarget > 12) {
-                			av += bayWType.getMedAV();
-                        	} else {
-                			av += bayWType.getShortAV();
-                        	}
+                            threat = 12;
```

Comment:
Normally I don't bother about white space formatting, but it looks like the tabs got completely clobbered here.

Reply:
OK, autoindented it.